heron/common/src/cpp/metrics/tmaster-metrics.cpp

```diff
@@ -35,7 +35,9 @@
 namespace heron {
 namespace common {
 
-TMasterMetrics::TMasterMetrics(const sp_string& sinks_filename, EventLoop* eventLoop) {
+using std::shared_ptr;
```

Comment:
the others use std::shared_ptr and here "using" is used. Better to be consistent.